ClientApp/src/Pages/GettingStarted/GettingStarted.scss

```diff
@@ -45,13 +45,32 @@
 }
 
 @media only screen and (max-width: 1000px) {
```

Comment:
for these max-width 1000px, and min width 1001px there are constants that yvonne and i used and defined on home.scss you might want to use them too

Reply:
actioned :)